src/plugins/workspace/public/components/workspace_form/workspace_use_case.tsx

```diff
@@ -79,20 +118,18 @@ export const WorkspaceUseCase = ({
       fullWidth
     >
       <EuiFlexGroup>
-        {availableUseCases
-          .filter((item) => !item.systematic)
-          .concat(DEFAULT_NAV_GROUPS.all)
-          .map(({ id, title, description }) => (
-            <EuiFlexItem key={id}>
-              <WorkspaceUseCaseCard
-                id={id}
-                title={title}
-                description={description}
-                checked={value === id}
-                onChange={onChange}
-              />
-            </EuiFlexItem>
-          ))}
+        {displayedUseCases.map(({ id, title, description, disabled }) => (
+          <EuiFlexItem key={id}>
+            <WorkspaceUseCaseCard
+              id={id}
+              title={title}
+              description={description}
+              checked={value === id}
+              onChange={onChange}
+              disabled={disabled}
+            />
+          </EuiFlexItem>
+        ))}
```

Comment:
can't find the place set `disabled`, does we still need it? I see we are doing filter instead of set other use case disabled.

```typescript
// When creating and isOnlyAllowEssential is true, only display essential use case
    if (isOnlyAllowEssential && operationType === WorkspaceOperationType.Create) {
      allAvailableUseCases = allAvailableUseCases.filter(
        (item) => item.id === DEFAULT_NAV_GROUPS.analytics.id
      );
    }
    setDisplayedUseCases(allAvailableUseCases);


```

Reply:
Oh, I missed to delete it.Thanks for catching it.

Reply:
Updated.